sklearn/model_selection/tests/test_search.py

```diff
@@ -152,7 +155,8 @@ def test_grid_search():
     assert_equal(grid_search.best_estimator_.foo_param, 2)
 
     for i, foo_i in enumerate([1, 2, 3]):
-        assert_true(grid_search.grid_scores_[i][0] == {'foo_param': foo_i})
+        assert_true(grid_search.results_["param_foo_param"][i] == foo_i)
```

Comment:
You can do this without looping with the new interface. Also the tests should be kept separate..